File: challenges/4-fizz-buzz/php/kayehol/fizzbuzz.php
<?php
$valorMax = intval($argv[1]);
if (!$valorMax) {
    echo "[Argumento inválido] : Insira um número".PHP_EOL;
    die();
}
for ($valorAtual = 1; $valorAtual <= $valorMax; $valorAtual++) {
    if ($valorAtual % 3 == 0 && $valorAtual % 5 == 0) {
        echo "FizzBuzz".PHP_EOL;
    } else if ($valorAtual % 3 == 0) {
        echo "Fizz".PHP_EOL;
    } else if ($valorAtual % 5 == 0) {
        echo "Buzz".PHP_EOL;
    } else {
        echo $valorAtual.PHP_EOL;
    }
}

File: challenges/4-fizz-buzz/python/uKaigo/fizzbuzz.py
n = int(input('Insira um número: '))

print()

for i in range(1, n + 1):
    output = ''

    if i % 3 == 0:
        output += 'Fizz'
    if i % 5 == 0:
        output += 'Buzz'
    if not output:
        output += str(i)

    print(output)
